tools/fast_nvcc/fast_nvcc.py

```diff
@@ -275,6 +282,8 @@ def is_weakly_connected(graph):
     """
     Return true iff graph is weakly connected.
     """
+    if (graph is None or len(graph) == 0):
+        return False
```

Comment:
Similar question here as above: it'd be great if you could give an example showing the need for this change

Reply:
I would be inclined to say that the null graph is actually connected here:
```suggestion
    if not graph:
        return True
```
The advantage of doing this is that if an `nvcc` invocation produces no `--dryrun`-esque output, I think this would result in it just passing straight through without error. Is there a reason to return `False` here instead of `True`?